convert2rhel/subscription.py

```diff
@@ -793,7 +793,7 @@ def _relevant_subscription_manager_pkgs():
     """
     relevant_pkgs = [
         "subscription-manager",
-        "subscription-manager-rhsm-certificates",
```

Comment:
Only thing that worries me is this, does this create am impact on el7 or el8 conversions?

Reply:
I can add a conditional to add the non versioned package for el7 and el8